gitea/apiobject.py

```diff
@@ -575,6 +576,65 @@ def delete(self):
         )
         self.deleted = True
 
+    def migrate_repo(
+            self,
+            service: str,
+            clone_addr: str,
+            repo_name: str,
+            description: str = "",
+            private: bool = False,
+            auth_token: str = None,
+            auth_username: str = None,
+            auth_password: str = None,
+            mirror: bool = False,
+            mirror_interval: str = None,
+            lfs: bool = False,
+            lfs_endpoint: str = "",
+            wiki: bool = False,
+            labels: bool = False,
+            issues: bool = False,
+            pull_requests: bool = False,
+            releases: bool = False,
+            milestones: bool = False,
+            repo_owner: str = None
+    ):
+        """ Migrate a Repository from another service.
+
+        Throws:
+            AlreadyExistsException: If the Repository exists already.
+            Exception: If something else went wrong.
+        """
+        result = self.gitea.requests_post(
+            Repository.REPO_MIGRATE,
```

Comment:
Just a note a not a requirement to change, but to easy sub classing it would be great to refer to `self` and not `Repository`. I know using the class name is prevalent, but going forward I'd hope to see more `self`.